apps/example/macos/Podfile

```diff
@@ -0,0 +1,13 @@
+ws_dir = Pathname.new(__dir__)
+ws_dir = ws_dir.parent until
+  File.exist?("#{ws_dir}/node_modules/react-native-test-app/macos/test_app.rb") ||
+  ws_dir.expand_path.to_s == '/'
+require "#{ws_dir}/node_modules/react-native-test-app/macos/test_app.rb"
+
+workspace 'ReactNativeBottomTabs.xcworkspace'
+
+use_test_app! do |test_app|
+  # Workaround for not using use_frameworks! in the Podfile
+  pod 'SDWebImage', :modular_headers => true
+  pod 'SDWebImageSVGCoder', :modular_headers => true
```

Comment:
Are these because expo-image uses them?

Reply:
Bottom tabs use them, it's a workaround for not needing to use static_frameworks - in the installation instructions I recommend to use static frameworks (or people could also enable modular headers globally but I had some issues with it). 

It's about cocoapods and mixing Swift with Objective-C